.github/scripts/check_advanced_requirement.sh

```
@@ -0,0 +1,48 @@
+
+echo "Stats for @$ASSIGNEE: GFI: $GFI_COUNT, Intermediate: $INT_COUNT"
+
+# 3. Validation Logic
+if (( GFI_COUNT >= 1 )) && (( INT_COUNT >= 1 )); then
+  echo "Qualification met."
+  exit 0
+else
+  echo "Qualification failed. Revoking assignment."
+
+  # Remove the assignee
+  gh issue edit $ISSUE_NUMBER --repo $REPO --remove-assignee "$ASSIGNEE"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)